typekits/rtt_kdl_conversions/package.xml

```diff
@@ -4,7 +4,7 @@
   <description>
     This package contains the components of the kdl_conversions package
   </description>
-  <license>GPL</license>
+  <license>BSD</license>
   <maintainer email="orocos-dev@lists.mech.kuleuven.be">Orocos Developers</maintainer>
```

Comment:
Unrelated to licensing, but this email address does not exist anymore. I suggest to update it to `orocos-dev@orocos.org` in all packages, but we likely should designate a dedicated person or a smaller group of people in a follow-up PR.